bin/handler-pagerduty.rb

```diff
@@ -20,19 +20,26 @@
 # Pagerduty
 #
 class Pagerduty < Sensu::Handler
+  option :json_config,
+         description: 'Config Name',
+         short: '-j JsonConfig',
+         long: '--json_config JsonConfig',
+         required: false
+
   def incident_key # rubocop:disable all
     source = @event['check']['source'] || @event['client']['name']
     [source, @event['check']['name']].join('/')
   end
 
   def handle # rubocop:disable all
+    json_config = config[:json_config] || 'pagerduty'
```

Comment:
I'd suggest removing this and using a default on the option itself - this'd then show in the `--help` output form the script and helps ensure the configuration is correct

```
  option :json_config,
         description: 'Config Name',
         short: '-j JsonConfig',
         long: '--json_config JsonConfig',
         required: false
+         default: 'pagerduty'
```